test/spec/modules/luponmediaBidAdapter_spec.js

```diff
@@ -364,4 +364,34 @@ describe('luponmediaBidAdapter', function () {
       expect(checkSchain).to.equal(false);
     });
   });
+
+  describe('On winning bid', function () {
+    const bids = {
+      'bidderCode': 'luponmedia',
+      'width': 300,
+      'height': 250,
+      'statusMessage': 'Bid available',
+      'adId': '105bbf8c54453ff',
+      'requestId': '934b8752185955',
+      'mediaType': 'banner',
+      'source': 'client',
+      'cpm': 0.364,
+      'creativeId': '443801010',
+      'currency': 'USD',
+      'netRevenue': false,
+      'ttl': 300,
+      'referrer': '',
+      'ad': '',
+      'auctionId': '926a8ea3-3dd4-4bf2-95ab-c85c2ce7e99b',
+      'responseTimestamp': 1598527728026,
+      'requestTimestamp': 1598527727629,
+      'bidder': 'luponmedia',
+      'adUnitCode': 'div-gpt-ad-1533155193780-5',
+      'timeToRespond': 397,
+      'size': '300x250',
+      'status': 'rendered'
+    };
+
+    spec.onBidWon(bids);
```

Comment:
This does not actually test anything.

See this https://github.com/prebid/Prebid.js/blob/master/test/spec/modules/adpartnerBidAdapter_spec.js#L199 example.

From Line 199 and on.

You can see they trigger the `spec.onBidWon` but are validating the return, as well as stubbing the request and making sure it has the right data they expect.

Reply:
Hi @robertrmartinez we just made the change that you suggested.